.travis.yml

```diff
@@ -35,11 +35,10 @@ deploy:
   provider: puppetforge
   user: puppet
   password:
-    secure: "A7lnigQt5QnJQfafRBPEQo1xSmEarUMZtw16SdBkQFSrMb/PJuihrIHwEUOzYK1lhXArxJQ5fB1WW1QYGaVCygewQNOgUje2dUER0DyuphWoJf1woun7p5VbZE1q0SV1U8wmDitViUe1iC7q9rFTaA3oMfVUxQaD4IsrCOTQqcE="
```

Comment:
hmm.. looks like that's already in `.sync.yaml`? not sure why it would have been removed .. I'll have a look